docs/change_log/release-3.3.md

```diff
@@ -102,6 +102,7 @@ The following bug fixes are included in the 3.3 release:
 * Fix complex scenarios involving lists and admonitions (#1004).
 * Fix complex scenarios with nested ordered and unordered lists in a definition list (#918).
 * Fix corner cases with lists under admonitions.
+* Automail does not treat missing local-part or domain as a valid email addresses (#1165).
```

Comment:
Version 3.3 has already been released. This note does not go here. Instead it should be added to the notes for 3.3.5 at [`docs/change_log/index.md`](https://github.com/Python-Markdown/markdown/blob/master/docs/change_log/index.md)

Reply:
Also, they are called "[Automatic Links][1]", not "Automail". Changing that will address the [failing spellchecker test][2]. You can ignore the failing `checklinks` test as it is broken.

[1]: https://daringfireball.net/projects/markdown/syntax#autolink
[2]: https://github.com/Python-Markdown/markdown/pull/1165/checks?check_run_id=3224274586